packages/test/src/test/RepositoryClient.ts

```diff
@@ -29,62 +29,48 @@ export class RepositoryClient {
     }
 
     async testPartitions() {
-        console.log(`test.partitions`)
+        // console.log(`test.partitions`)
```

Comment:
Should remove this and the other commented out lines in this file?

Reply:
Yes